plotting/map.py

```diff
@@ -344,6 +344,7 @@ def load_data(self):
 
         if self.compare:
             self.variable_name += " Difference"
+            self.cmap = cmap = colormap.find_colormap(self.compare.get('colormap_diff'))
```

Comment:
I looked further down the file and I can see why you've got the chained assignment thing going on here. What you've is fine for now.

Reply:
I did not even know you could do that, or that I had done it.
Should I i get rid of the cmap assignment? lol

Reply:
LOL well I think it's needed since  `cmap` (not `self.cmap`) is referenced below here.